### ISSUE TITLE
How to accept socket.io connection from an android client in Sails server?

### ISSUE THREAD

**Issue by yedhukrishnan** [balderdashy/sails#2349]
We have an android client which sends a socket connect request to the Node JS server. While using **Express JS**, this is the code which accepts the connection (copied from [chat application example](http://socket.io/get-started/chat/)):

    io.on('connection', function(socket){
      console.log('a user connected');
    });

This is working as expected and we are getting 'a user connected' log in the console. But when we try to accept the connection request in our **Sails JS** server, it is not working. In Sails, we are using `sails.io`. Here is the code:

    sails.io.on('connection', function(socket) {
      console.log('a user connected');
    });

We added the above code in the `app.js` file. But it was not working. Is this the right way to accept a connection in a Sails server? If not, how do we do that?

**Comment by loicsaintroch** [comment]
Any update about this issue?

**Comment by LeonardoMinati** [comment]
I have the same problem now, did someone solved it?

**Comment by sailsbot** [comment]
Thanks for posting, @yedhukrishnan.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by xhb** [comment]
Any update about this issue? i need it now...
